src/Raygun.Blazor/Logging/IRaygunLogger.cs

```diff
@@ -0,0 +1,35 @@
+
+namespace Raygun.Blazor.Logging
+{
+    /// <summary>
+    /// Raygun logger interface.
+    /// </summary>
+    public interface IRaygunLogger
```

Comment:
This feels awfully close to an `ILogger` interface. If you make this public, developers might be confused. Consider making this internal.